keel-core/src/main/kotlin/com/netflix/spinnaker/keel/tags/EntityTag.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.keel.tags
+
+data class EntityTag(
```

Comment:
It would be great to have some doc comments explaining what these classes are for, I don't find them very intuitive to understand.

Reply:
Adding, and removing unneeded ones!